plexpy/graphs.py

```diff
@@ -854,6 +854,103 @@ def get_total_plays_per_stream_type(self, time_range='30', y_axis='plays', user_
                   'series': [series_1_output, series_2_output, series_3_output]}
         return output
 
+    def get_total_concurrent_streams_per_stream_type(self, time_range='30'):
+        monitor_db = database.MonitorDatabase()
+
+        time_range = helpers.cast_to_int(time_range) or 30
+        timestamp = helpers.timestamp() - time_range * 24 * 60 * 60
+        
+        def calc_most_concurrent(result):
```

Comment:
Could be moved to `helpers` and be used both in `graphs` and `datafactory`. Beeing the only difference the `title` property in the `datafactory` version.

Reply:
Decided against moving it in `helpers` in favor of optimizing the local version by removing not required informations.